storage/common/include/algebra/storage/vector.hpp

```diff
@@ -130,9 +142,7 @@ class vector {
   /// Sets the trailing uninitialized values to zero.
   template <std::size_t... Is>
   constexpr void zero_fill(std::index_sequence<Is...>) noexcept {
-    if constexpr (sizeof...(Is) > 0) {
-      //((m_data[N - sizeof...(Is) + Is] = value_t(0)), ...);
-    }
+    //((m_data[N - sizeof...(Is) + Is] = value_t(0)), ...);
```

Comment:
I don't remember the story here. Why don't we actually set the trailing values to zero?

Reply:
Because Vc emits a warning that an initialization with a comma operator is deprecated and I am not sure how to ignore that warning so that it would pass the CI?

Reply:
:weary: Okay...

Reply:
yeah, in later versions of this code, I set the trailing zeroes by hand according to the two vector sizes where we actually need this: 3D and 6D vectors